File: testprojects/java/benchmark/accuracy-test-2013-04-17/domain/direct/violating/AccessObjectReferenceWithinIfStatement.java
package domain.direct.violating;

import domain.direct.Base;

public class AccessObjectReferenceWithinIfStatement extends Base{
	
	public AccessObjectReferenceWithinIfStatement(){
		if (profileDao != null) {
			String s = "wrong";
			System.out.println(s);
		}
	}
}

File: testresources/csharp/benchmark/Domain/Direct/Allowed/AccessObjectReferenceWithinIfStatement.cs
namespace Domain.Direct.Violating
{
	using Domain.Direct;	

	public class AccessObjectReferenceWithinIfStatement : Base
	{
		public AccessObjectReferenceWithinIfStatement()
		{
			if (profileDao != null)
			{
                string s = "wrong";
			}
		}
	}
}
